Header header
float32 distance_left
float32 distance_right
float32 distance_floor
float32 distance_ceiling
float32 distance_front
float32 pitch
float32 roll
float32 yaw
